src/NativeTypes/Primitives.tt

```diff
@@ -408,6 +409,8 @@ namespace System
 			return ((IConvertible)v).ToType (targetType, provider);
 		}
 
+		[Obsolete ("This API is not available in .NET, use Buffer.MemoryCopy instead.")]
+		[EditorBrowsable (EditorBrowsableState.Never)]
```

Comment:
Is there a `#if !NET` somewhere that makes them not available in the .net profile ?

Reply:
@spouliot We're going to remove ObjCRuntime.nfloat and use System.Runtime.InteropServices.NFloat instead, and that type doesn't have these APIs.